RocketChat/Rocket.Chat
# [IMPROVE] Save E2EE password redesign

## Proposed changes (including videos or screenshots)


<img width="633" alt="Screenshot 2022-04-01 at 10 11 53 PM" src="https://user-images.githubusercontent.com/58601732/161307205-bebec0c5-c25b-44f6-b952-4c2e28fed82e.png">



## Issue(s)


## Steps to test or reproduce

- Go to My Account -> Security.
- Reset E2EE Key.
- Log back in and this save E2EE modal should appear.
## Further comments
